src/frontend/apps/impress/src/stores/useResponsiveStore.tsx

```diff
@@ -5,18 +5,21 @@ export type ScreenSize = 'small-mobile' | 'mobile' | 'tablet' | 'desktop';
 export interface UseResponsiveStore {
   isMobile: boolean;
   isTablet: boolean;
+  mobileMenuIsOpen: boolean;
   isSmallMobile: boolean;
   screenSize: ScreenSize;
   screenWidth: number;
   setScreenSize: (size: ScreenSize) => void;
   isDesktop: boolean;
+  toggleMobileMenu: () => void;
```

Comment:
I don't think we should put this logic in this store, `useResponsive` should not have a menu logic in it, it is part of the `LeftPanel`.

Reply:
For me it is absolutely linked to responsive. Already, we only use it when you are responsive. 

In addition if we want for example to close or not depending on the screen resizing. 

If we have to create another blind and depending on the changes use this blind, I would find it a shame to have 2 renderings instead of one.